### PR TITLE
Fixed the bug in the filtering of multiple PacBio files

### LINKED ISSUE #77: Fix when multiple PacBio files are given
### Description of the bug

The alignment subworkflow was imported from readmapping pipeline.
Import changes from this bug fix PR https://github.com/sanger-tol/readmapping/pull/90 to here

### Command used and terminal output

_No response_

### Relevant files

_No response_

### System information

_No response_

### PR DESCRIPTION
**Pull request by gq1**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/sanger-tol/variantcalling/tree/main/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by gq1** [comment]
I copied all the changes from readmapping PR to here https://github.com/sanger-tol/readmapping/pull/90

This PR being blocked because we are trying to merge to protected main branch directly. This is a bug fix so it will be OK and we will force to merge.

The readmpping pipeline CI set the protection branch as master, that is why the block check not being done.
https://github.com/sanger-tol/readmapping/blob/a27e9ce841d230f3c1ca5086019a134a116bee18/.github/workflows/branch.yml#L6

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [[1.2.2](https://github.com/sanger-tol/readmapping/releases/tag/1.1.3)] - Shang Tang (patch 2) - [2024-05-24]
```
Comment (muffato) [inline CHANGELOG.md:6]:
```suggestion
## [[1.1.3](https://github.com/sanger-tol/readmapping/releases/tag/1.1.3)] - Shang Tang (patch 2) - [2024-05-24]
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,12 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [[1.1.3](https://github.com/sanger-tol/readmapping/releases/tag/1.1.3)] - Shang Tang (patch 2) - [2024-05-24]
```
Comment (muffato) [inline CHANGELOG.md:6]:
```suggestion
## [[1.1.3](https://github.com/sanger-tol/readmapping/releases/tag/1.1.3)] - Shang Tang (patch 3) - [2024-05-24]
```

**Review by muffato — APPROVED** [review]
We're not good with typing text today :) but we got there in the end !

**Pull request merged**